src/speculators/convert/eagle/eagle3_converter.py

```diff
@@ -0,0 +1,170 @@
+"""
```

Comment:
Let's move this outside of the eagle folder, maybe in it's own folder inside convert?